ckanext/privatedatasets/auth.py

```diff
@@ -137,3 +137,7 @@ def package_acquired(context, data_dict):
 def acquisitions_list(context, data_dict):
     # Users can get only their acquisitions list
     return {'success': context['user'] == data_dict['user']}
+
+def  package_deleted(context, data_dict):
+    # TODO: Check functionality and improve security(if needed)
```

Comment:
Technically, the function is protected by Apache with SSL client verification: only those clients with the appropriate certificate are able to call this function. That's why this function always returns true. Please read the "Securing the Notification Callback" section from README.md .